<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Riya Dua - Website</title>
  <link rel="stylesheet" href="styles.css">
</head>
<body>
  <header>
    <h1>Riya Dua</h1>
<h3> A passionate Bioinformatician</h3>
  
  <nav>
    <ul>
      <li><a href="index.html">Home</a></li>
      
      <li><a href="publications.html">Publications</a></li>
      <li><a href="contact.html">Contact</a></li>
    </ul>
  </nav>
</header>
  <main>
    <section id="home">
      <h2>Home </h2>
      <p>

Hi, I am Riya Dua, pursuing MS in Bioinformatics from Johns Hopkins University, Baltimore, MD, USA. Welcome to My Professional Journey! Here, I'm excited to share with you my journey through various projects and contributions in the field of Bioinformatics and Biological Data Analysis. </p>

<p>Explore the projects I've been involved in, ranging from NGS Data Analysis to Computational Molecular Drug Designing, where I've utilized cutting-edge tools and techniques such as High Performing Computing Clusters, Machine Learning, Network Biology, Bioconductor Packages in R and so on. Each project represents a step forward in my commitment to enhance healthcare.</p>
</p>

<p>I am dedicated to continuous learning and have earned certifications in Data Structures and Algorithms, DNA sequencing algorithms, Data Science, and Data Analytics. These certifications complement my academic achievements and practical research skills.</p>

<p>Feel free to navigate through the different sections to learn more about my background, explore detailed project descriptions, and reach out to discuss potential opportunities.</p>

Thank you for visiting, and I look forward to connecting with you!</p>
    </section>

  <section id="projects">

    <h2>Projects</h2>
    <div class="project-box">
<div class="project-box-content">

      <h3>Charted the Epigenetic Dynamics in Colon Carcinogenesis using Chip-Seq Analysis techniques</h3>
<details>
<p> Tools Used: </p>
<p>>Upstream analysis on Linux: FASTQC, Bowtie2, MACS2, Shell, HPC (slurm)</p>
<p>>Downstream analysis: R (DiffBind) 
<summary>Read More</summary>
      <p>Utilizing Chip-Seq data analysis to map the epigenetic changes associated with BRAF V600E mutations in colon cancer. </details></p>
    </div><img src="images/chip.png" alt="chip"></details></div>

    <div class="project-box">
  <div class="project-box-content">


      <h3>Identification of Oral Cancer Candidate Genes using Machine Learning Approaches</h3>
<details>
<p> Tools Used: </p>
<p>>Upstream analysis on Linux: FASTQC, STAR Alignment, HPC </p>
<p>>Downstream analysis: R (DESeq2, ggplot2, tidyverse, dplyr, WGCNA, ML-seq), Cytoscape (Network Analysis), Gene Enrichment Analysis</p> 
<summary>Read More</summary>
      <p>This project was my Master's (Biotechnology) Thesis project where we aimed to develop a machine learning model that identifies Oral cancer candidate genes. A publically available dataset was taken and rigorous data preparation, data cleaning, and outlier filteration was performed in order to build a highly accurate model. This model was built using MLseq package in R and used DEGs as the features to train the model. These DEGs were obtained using RNA-seq analysis and Network Analysis was performed by Weighted Gene Coexpression Network Analysis where significnat gene modules in oral cancer were identified. Finally, SVM classification, with the deseq-vst transformation-normalization technique performed the best out of all the testing models giving the accuracy of 99%. This project is currently under works to be submitted for publication.</p>
</div>
<img src="JNU/Dissertation.png" alt="Dissertation"></details>
    </div>

    <div class="project-box">
<div class="project-box-content">
                <h3>Pre-Term Birth Biomarker Identification for Human Microbiome using Metabolomics and Machine Learning Approaches</h3>
<details>
<p> Tools Used: Python (scikit-learn, matplotlib, NumPY, pandas, Pyplot) </p>
<summary>Read More</summary>                
<p>In this project, we developed a Machine Learning-Infused Metabolomics Model for Preterm Birth Prediction in Humans. Raw data files were taken from EMBL and meticulous data cleaning and data preparation was performed. The raw data contained 455 samples and more than 300 metabolites out of which 53 metabolites were considered significant in occurance of PTB and thus, they were taken as features for out ML model followed by train-test split. Five algorithms were used in the evaluation of this study for comparative study and to deduce the best algorithm for prediction based on F1 score, AUC value, recall score and accuracy. The models were tested on labelled and unlabelled data. Out of the five models Gradient Boosting and Decision Trees perfromed the best with the accuracy of 97%. And finally, to understand the importance of each feature, a feature importance plot was used. This plot helps in the visualization and quantification of the relative importance of different features in the predictive model.  View the publication <a href="https://ieeexplore.ieee.org/abstract/document/10307581"> here</a></p>
</div><img src = "Amity/ML.PNG" alt="Pre-term Birth"></details></div>


<div class="project-box">
<div class="project-box-content">
                <h3>Omics-based Analysis of Metabolic Genes, Proteins, and Pathways for Antibacterial Target Identification in Bacterial Vaginosis</h3>
<details>
<p> Tools Used: Parallel-Meta 3 (Linux), AutoDock, PyMOL, R </p>
<summary>Read More</summary>
                <p>Utilized omics data to identify potential antibacterial targets in the context of bacterial vaginosis. 16S rRNA raw vaginal microbiome samples of women stratified by vaginal health condition was taken from a public source. This raw sequencing data underwent taxonomic classification and functional profiling to determine the microbial and functional composition of the vaginal microbiome samples, using the Parallel-Meta 3 software. The genes identified in control and affected samples were compared and genes exclusive to BV were identified which were then chosen for molecular docking. Inhibitors for these proteins were chosen from DrugBank and PubChem. Following which the team performed molecular dynamic simulation and ADMET analysis. This finally revealed Dalfopristin, Clorgyline, and Hydrazine as potential therapeutics for the treatment of bacterial vaginosis. View the publication <a href="https://www.sciencedirect.com/science/article/pii/S0732889324001780#sec0002">here</a></p>
                
            </div><img src="Amity/Pre-Term Birth.PNG" alt="Bacterial Vaginosis"></details></div>


            

            <div class="project-box">
<div class="project-box-content">
                
<h3>Identified Hub Genes and Their Crucial Pathways for Survival in Cervical Cancer Patients Utilizing In-Silico Approaches</h3>
<details>
<p> Tools Used: R (DESeq2, GEOquery, tidyr, limma, pheatmap, dplyr), Cytoscape for Network Analysis, AutoDock, GROMACS, PyMOL </p>
<summary>Read More</summary>                
<p>To identify the hub genes involved in cervical cancer, microarray profiling data was taken from NCBI-GEO database and was preprocessed in away that it can be analysed by DESeq2 package. DEGs were identified using DESeq2 and limma along with volcano plot and heatmap visualizations. Protein-protein interaction was studied further us=tilizing STRING database and cytoscape using which top 10 genes were identified that play a significant role in causing the cervical cancer. For the proteins of these genes, molecular docking was performed with potential phytoconstituents taken from PubChem, using AutoDock which underwent molcular dynamic simulation to investigate stability of protein-ligand complexes.    </p>
                
            </div><img src = "JNU/Cervicalcancer.PNG" alt="cervical cancer hub genes"></details></div>


            <div class="project-box">
<div class="project-box-content">
                <h3>Computer-Aided Drug Designing against OMRL-1 Like Protein 3 Involved in Asthma</h3>
<details>
<p> Tools and Databases Used: NCBI, BLAST, Modeller, DogSite,  AutoDock, GROMACS (Linux), PubChem </p>
<summary>Read More</summary>                 
<p>OMRDL3 gene is a key player in Aasthma condition and thus, this was selected and my aim was to design and identify drugs that inhibit the expression of this gene. Protein was selected after litrary search and its 3D structure was selected from PDB. Ran alignment of the protein to get top 10 best matches. Through literature search found drugs that can potentially interact with out selected protein and other aligned proteins. Analyzed 150+ drug SDF files identifying the top 10 candidates for Asthma; executed Docking and Simulation for 10 drug-protein pairs with AutoDock and ran molecular dynamic simulations on GROMACS to test their stability and select the best pair.
 </p>
                
            </div><img src="images/training.PNG" alt="Asthma protein"></details></div>

<div class="project-box">
<div class="project-box-content">
                <h3>Structural and Functional Consequences of Non-Synonymous SNPs in the Human LGALS-1 Gene</h3>
<details>
<p> Tools and Databases Used: NCBI, BLAST, i-mutant, predict-snp, i-TASSER, HOPE, GROMACS (Linux) </p>
<summary>Read More</summary>                 
<p>Found significant SNPs in LGALS-1 causing mayocardial infraction using sequence based and structure based predictions using online tools. Studied the snps which degrade the proteins, the difference between snps and the actual gene (wild type) on the bases of size, hydrophobicity and charge. Identified 22 significant SNPs, performed conservation analysis, protein modeling in PyMOL and simulated with the changed sequence-snp using protein-in-water GROMACS. </p>
                
            </div></details></div>
  </section>
<section>
            <h2>Career Goals</h2>
            <p>My professional journey is driven by a keen interest in exploring advanced data analysis techniques and harnessing the power of machine learning to advance healthcare and personalized medicine. With a background in bioinformatics and data analysis, I am passionate about leveraging these disciplines to uncover actionable insights from complex biological datasets.</p>

I am particularly enthusiastic about enhancing my proficiency in sophisticated data analysis methodologies, including the integration of genomic, transcriptomic, and proteomic data with machine learning. By employing these techniques, I aim to unravel intricate biological mechanisms underlying diseases and develop novel approaches for diagnosis, treatment, and prevention.</p>


Central to my career vision is the impact on healthcare delivery and personalized medicine. I am committed to contributing to transformative advancements in these fields, translating research findings into practical applications that improve patient outcomes globally. I envision playing a role in shaping the future of healthcare by bridging the gap between computational insights and clinical practice.</p>

Continuous learning is fundamental to my professional growth. I am dedicated to staying abreast of emerging technologies and methodologies in bioinformatics and machine learning. Collaborating with multidisciplinary teams is essential to me, as it fosters innovation and enables holistic approaches to solving complex healthcare challenges.</p>
        </section>


  </main>
  <footer>
    <p>&copy; Riya Dua - Website</p>
Contact: rdua5@jh.edu, riyadua99@gmail.com
  </footer>
  
</body>
</html>
